Apromore-Custom-Plugins/Process-Simulation-Info-Logic/src/main/java/org/apromore/processsimulation/service/SimulationInfoService.java

```diff
@@ -204,14 +206,18 @@ private void deriveTimetable(
 
         CalendarModel calendarModel = simulationData.getCalendarModel();
 
-        if (calendarModel.getName().equals(SimulationData.DEFAULT_CALENDAR_NAME)) {
```

Comment:
Likely can get rid of the Constant SimulationData.DEFAULT_CALENDAR_NAME

Reply:
Keeping this one SimulationData.DEFAULT_CALENDAR_NAME. Remove the one i have created.